docs/program/apis/cloud.md

```diff
@@ -756,3 +890,445 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
+- `resource_type` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The type of the resource to add the role to (either `"organization"`, `"location"`, or `"robot"`).
+  Must match the type of the `resource_id`'s resource.
+- `resource_id` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the resource the role applies to (the ID of either an {{< glossary_tooltip term_id="organization" text="organization" >}}, {{< glossary_tooltip term_id="location" text="location" >}}, or {{< glossary_tooltip term_id="robot" text="robot" >}}.)
+
+**Raises:**
+
+- `GRPCError`: This error is raised if an invalid identity ID, role, resource type, or resource ID is passed.
+
+```python {class="line-numbers linkable-line-numbers"}
+await cloud.add_role(
+  identity_id="abc01234-0123-4567-ab12-a11a00a2aa22",
+  role="owner",
+  resource_type="location",
+  resource_id="111ab12345")
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.add_role).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### RemoveRole
+
+Remove a role under the organization you are currently authenticated to.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `identity_id` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the entity the role belongs to (for example, a user ID).
+- `role` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The role to remove (either `"owner"` or `"operator"`).
+- `resource_type` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The type of the resource to remove the role from (either `"organization"`, `"location"`, or `"robot"`).
+  Must match the type of the `resource_id`'s resource.
+- `resource_id` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the resource the role applies to (the ID of either an {{< glossary_tooltip term_id="organization" text="organization" >}}, {{< glossary_tooltip term_id="location" text="location" >}}, or {{< glossary_tooltip term_id="robot" text="robot" >}}.)
+
+**Raises:**
+
+- `GRPCError`: This error is raised if an invalid identity ID, role, resource type, or resource ID is passed.
+
+```python {class="line-numbers linkable-line-numbers"}
+await cloud.remove_role(
+  identity_id="abc01234-0123-4567-ab12-a11a00a2aa22",
+  role="owner",
+  resource_type="location",
+  resource_id="111ab12345")
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.remove_role).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### ListAuthorizations
+
+List all authorizations under a specific resource (or resources) within the organization you are currently authenticated to.
```

Comment:
I see how this sentence could be interpreted two different ways because of its ambiguous structure. But I think that, like the other usages of "currently authenticated to" in this document, "authenticated" is correct. Changing it to "authorized for" would make the sentence circular and incorrect I think, because it's listing all authorizations, not just your own. For example, if I run it with no params while authenticated to one of my orgs, it shows that i'm an owner and it shows that another person is an operator.
```suggestion
List all authorizations (owners and operators) under a specific resource (or resources) within the organization you are currently authenticated to.
```
Would this be better?

Reply:
This should also become more clear once we sort out the "resource" language.

Reply:
Yeah no your right I was wrong

Reply:
Should I commit my suggestion or does its grammatical weirdness make it not worth the possible clarity improvement?

Reply:
yeah suggestion looks good!